Author: Dave Jones
Forwarded: not-needed
Description: Fix privacy violation warning in local documentation
 Lintian doesn't permit locally hosted documentation to refer to remote hosts.
 The JSON schema documentation included in the package requires JQuery and
 Bootstrap. This patch simply modifies the HTML to refer to locally installed
 copies (and removes the pretty, but remote, font references).

--- a/doc/json-schema/os-list-schema.html
+++ b/doc/json-schema/os-list-schema.html
@@ -1,4 +1,4 @@
-<!DOCTYPE html><html lang=en> <head><link rel=stylesheet type=text/css href="https://fonts.googleapis.com/css?family=Overpass:300,400,600,800"><script src=https://code.jquery.com/jquery-3.4.1.min.js integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin=anonymous></script><link href=https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css rel=stylesheet integrity=sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T crossorigin=anonymous><script src=https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js integrity=sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM crossorigin=anonymous></script><link rel=stylesheet type=text/css href=schema_doc.css><script src=https://use.fontawesome.com/facf9fa52c.js></script><script src=schema_doc.min.js></script><meta charset=utf-8><title>Imager oslist.json schema</title></head> <body onload=anchorOnLoad(); id=root><h1>Imager oslist.json schema</h1><div class=text-right> <button class="btn btn-primary" type=button data-toggle=collapse data-target=.collapse:not(.show) aria-expanded=false>Expand all</button> <button class="btn btn-primary" type=button data-toggle=collapse data-target=.collapse.show aria-expanded=false>Collapse all</button> </div> <div class=breadcrumbs></div><span class="badge badge-dark value-type">Type: object</span> <span class="badge badge-success default-value">Default: {}</span><br> <span class=description><p>oslist.json format.</p> </span> <span class="badge badge-info no-additional">No Additional Properties</span> <br> <div class="badge badge-secondary">Example:</div> <br><button class="btn btn-light example-show collapsed" data-toggle=collapse data-target=#root_ex1 aria-controls=root_ex1></button><div id=root_ex1 class="collapse jumbotron examples"><div class=highlight><pre><span></span><span class=p>{</span>
+<!DOCTYPE html><html lang=en> <head><script src=jquery.min.js crossorigin=anonymous></script><link href=bootstrap.min.css rel=stylesheet crossorigin=anonymous><script src=bootstrap.min.js crossorigin=anonymous></script><link rel=stylesheet type=text/css href=schema_doc.css><script src=schema_doc.min.js></script><meta charset=utf-8><title>Imager oslist.json schema</title></head> <body onload=anchorOnLoad(); id=root><h1>Imager oslist.json schema</h1><div class=text-right> <button class="btn btn-primary" type=button data-toggle=collapse data-target=.collapse:not(.show) aria-expanded=false>Expand all</button> <button class="btn btn-primary" type=button data-toggle=collapse data-target=.collapse.show aria-expanded=false>Collapse all</button> </div> <div class=breadcrumbs></div><span class="badge badge-dark value-type">Type: object</span> <span class="badge badge-success default-value">Default: {}</span><br> <span class=description><p>oslist.json format.</p> </span> <span class="badge badge-info no-additional">No Additional Properties</span> <br> <div class="badge badge-secondary">Example:</div> <br><button class="btn btn-light example-show collapsed" data-toggle=collapse data-target=#root_ex1 aria-controls=root_ex1></button><div id=root_ex1 class="collapse jumbotron examples"><div class=highlight><pre><span></span><span class=p>{</span>
     <span class=s2>&quot;imager&quot;</span><span class=o>:</span> <span class=p>{</span>
         <span class=s2>&quot;latest_version&quot;</span><span class=o>:</span> <span class=s2>&quot;1.6&quot;</span><span class=p>,</span>
         <span class=s2>&quot;url&quot;</span><span class=o>:</span> <span class=s2>&quot;https://www.raspberrypi.org/software/&quot;</span><span class=p>,</span>
@@ -161,4 +161,4 @@
         <span class=s2>&quot;init_format&quot;</span><span class=o>:</span> <span class=s2>&quot;cloudinit&quot;</span>
     <span class=p>}</span>
 <span class=p>]</span>
-</pre></div> </div> </div> </div> </div> </div> <footer> <p class=generated-by-footer>Generated using <a href=https://github.com/coveooss/json-schema-for-humans>json-schema-for-humans</a> on 2022-02-03 at 13:40:39 +0100</p> </footer></body> </html>
\ No newline at end of file
+</pre></div> </div> </div> </div> </div> </div> <footer> <p class=generated-by-footer>Generated using <a href=https://github.com/coveooss/json-schema-for-humans>json-schema-for-humans</a> on 2022-02-03 at 13:40:39 +0100</p> </footer></body> </html>
